src/Antd_TimePicker.re

```diff
@@ -0,0 +1,8 @@
+type timePickerProps('a) = Js.t({. timePickerProps: int} as 'a);
```

Comment:
this is kinda hacky, could change it to just

```type timePickerProps('a) = Js.t('a);```

but then you lose some type safety as you could do

```<DatePicker showTime=TimePickerProps(someOtherSpecialCreationFunction(~foo="bar",())/>```

Reply:
type t is a convention in OCaml. It should be use instead. And I think this is not better than identity hack.

Reply:
how about 
```
module TimePickerProps {
  type t('a) = Js.t({. timePickerProps: int} as 'a);

  [@bs.obj]
  external makeTimePickerProps :
    (
      ~size: string=?,
      ~value: moment=?,
      ~defaultValue: moment=?, /* should be moment | moment[] */
      ~_open: Js.boolean=?,
      ~format: string=?,
      ~onChange: (moment, string) => unit=?,    
      ~onOpenChange: Js.boolean => unit=?,
      ~disabled: Js.boolean=?,
      ~placeholder: string=?,
      ~prefixCls: string=?,
      ~hideDisabledOptions: Js.boolean=?,
      ~disabledHours: unit => array(int)=?,
      ~disabledMinutes: int => array(int)=?,
      ~disabledSEconds: (int,int) => array(int)=?,
      ~getPopupContainer: Dom.element => Dom.htmlElement=?,
      ~addon: 'a=?,
      ~use12Hours: Js.boolean=?,
      ~focusOnOpen: Js.boolean=?,
      ~hourStep: Js.boolean=?,
      ~minuteStep: Js.boolean=?,
      ~secondStep: Js.boolean=?,
      ~allowEmpty: Js.boolean=?,
      ~clearText: Js.boolean=?,
      ~defaultOpenValue: moment=?,
      ~popupClassName: string=?,
      ~id: string=?,
      ~className: string=?,
      ~style: ReactDOMRe.Style.t=?,
      unit
    ) =>
    t(_) =
    "";
};```

Reply:
I don’t have an opinion on this actually. Both solutions are hacks so keeping the current type is fine with me. Anyway, is this type be reused elsewhere?

Reply:
Ya it is used in the showTime prop for DatePicker